projects/js-packages/charts/src/components/line-chart/stories/index.stories.tsx

```diff
@@ -359,78 +271,6 @@ BrokenLine.parameters = {
 	},
 };
 
-export const WithStartGlyphs: StoryObj< typeof LineChart > = Template.bind( {} );
```

Comment:
These were all moved to glyph.stories.tsx in https://github.com/Automattic/jetpack/pull/43922 but were reinstated by https://github.com/Automattic/jetpack/pull/43875, I assume due to a conflict.